### PR TITLE
doc/releases/octopus.rst: move still-pending release notes to official v15.2.0 release notes

### PR DESCRIPTION
**Pull request by smithfarm**
PendingReleaseNotes still had a bunch of release notes that were included in the Octopus v15.2.0 release notes. Delete these.

Also, apparently, several release notes that were added to `PendingReleaseNotes` for Octopus v15.2.0 did not actually make it into `doc/releases/octopus.rst` for the v15.2.0 release. Move these.

Finally, fix the heading in PendingReleaseNotes so it's clear the notes relate to Pacific.

### CODE REVIEW

**Comment by smithfarm** [comment]
@sebastian-philipp I was wondering, since all cephadm features are currently being immediately backported to Octopus and mentioned in the Octopus release notes, does it really make sense to also mention them in the Pacific release notes?

**Comment by sebastian-philipp** [comment]
> @sebastian-philipp I was wondering, since all cephadm features are currently being immediately backported to Octopus and mentioned in the Octopus release notes, does it make sense to also mention them in the Pacific release notes?

not really. Do you have an idea for a better workflow?

**Comment by smithfarm** [comment]
> not really. Do you have an idea for a better workflow?

Add them to directly to the `PendingReleaseNotes` file in the `octopus` branch?

File: PendingReleaseNotes [hunk 1]
```diff
@@ -1,324 +1,8 @@
->=15.0.0
+>=16.0.0
 --------
 
 * CVE-2020-10736: Fixes an authorization bypass in monitor and manager daemons
```
Comment (jdurgin) [inline PendingReleaseNotes:4]: this is in 15.2.2

Comment (smithfarm) [reply PendingReleaseNotes:4]: yeah, I guess CVEs fixed before 16.2.0 need not be mentioned in the Pacific release notes.

**Comment by smithfarm** [comment]
jenkins render docs

**Pull request merged**